packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetail.tsx

```diff
@@ -127,6 +128,18 @@ export const AlertDetail = () => {
             serviceType={alertDetails.service_type}
           />
         </Box>
+        <Box
+          sx={{
+            ...getAlertBoxStyles(theme),
+            overflow: 'auto',
+          }}
+        >
+          <AlertDetailNotification
+            channelIds={alertDetails.channels.map((channel) =>
+              Number(channel.id)
```

Comment:
is it necessary to type cast to number, can't we pass as it is because at the end we are going to convert it into comma separated string
```suggestion
             channel.id
```